tuxemon/locale.py

```diff
@@ -216,7 +216,8 @@ def replace_text(session: Session, text: str) -> str:
     text = text.replace("${{name}}", player.name)
     text = text.replace("${{currency}}", "$")
     text = text.replace(r"\n", "\n")
-    text = text.replace("${{money}}", str(player.money["player"]))
+    money = player.money.get("player", 0)
```

Comment:
why do we need to specify the player when we already have the player?

Reply:
this way seems more complicated.
admittedly, it's a marginal increase in complexity, but it's still an increase.

Is there a benefit to this method?

Reply:
because the money is a dict[str, int], where str is "player"

Reply:
why is it that way? i must have not noticed the change in a previous PR. is this to support multiple wallets or something?

Reply:
exactly, it supports multiple wallets